### ISSUE TITLE
Валидация fest:attributes и fest:attribute

### ISSUE THREAD

**Issue by RubaXa** [mailru/fest#2]
Вы не верно сделали валидацию данного атрибута, нужно чтоб он выдавал ошибку, если в нем есть любой другой tag, кроме  fest:attribute/

``` xml
<fest:attributes>
   <fest:attribute/>
   <any-tag />
</fest:attributes>
```

Также, fest:attribute/ не может быть находиться вне fest:attributes/

**Comment by AndrewSumin** [comment]
там могут быть другие теги, например fest:if для этого он и делался.

**Comment by RubaXa** [comment]
Не спорю, только взгляни ниже:

``` xml
<fest:attributes>
   <fest:attribute>
        <fest:attributes>
        </fest:attributes>
   </fest:attribute>
   <fest:value />
   <div>...</div>
</fest:attributes>
```

**Comment by AndrewSumin** [comment]
Тут согласен, но это другая постановка вопроса.

**Comment by RubaXa** [comment]
Согласен, не корректно написал, имелось ввиду, что нужно сделать корректный разбор того может находиться внутри и где они могу находиться, это я просто fest:attributes и fest:attribute
